doc/data.md

```diff
@@ -27,7 +27,7 @@ measures | String | This parameter specifies the measures to be returned. Possib
 includeHistory | Boolean | This parameter allows retrieving previous versions of the data, as they were disseminated in the past (*history* or *timeline* functionality). When the value is set to `true`, the returned data message should contain one or two datasets per data dissemination, depending on whether a dissemination also deleted observations from the data source. The `validFromDate` and/or `validToDate` attributes of the dataset should be used to indicate the periods of validity for the data contained in the data set. See below for an example on how to handle the `includeHistory` parameter. | `false` | No
 offset | Positive integer | The number of observations (or series keys) to skip before beginning to return observations (or series keys). | 0 | No
 limit | Positive integer | The maximum number of observations (or series keys) to be returned. If no limit is set, all matching observations (or series keys) must be returned. | | No
-sort | String | This parameter specifies the order in which the returned data should be sorted. Per component to be used for sorting, the syntax is the component ID, optionally followed by a colon and the sort order (`asc` or `desc`, with `asc` being the default). For example, to sort the data by descending time period: `sort=TIME_PERIOD:desc`. The plus (`+`) can be used whenever there is a need to sort by more than one component. For example, to sort the data by ascending INDICATOR and descending TIME_PERIOD: `sort=INDICATOR+TIME_PERIOD:desc`. Last but not least, the keyword `series_key` can be used to sort the data by series key. If the data has a time dimension and if TIME_PERIOD is not included in the sort parameter, then observations are grouped per time series and sorted within each time series by ascending time period. For any other component not included in the sort parameter, the related order is non-deterministic. Except for time periods, the sorting within a component is based on the code IDs or the non-coded component values. | | No
+sort | String | This parameter specifies the order in which the returned data should be sorted. It contains either one or more component IDs, by which the data should be sorted, separated by `+` (to indicate an AND), the `*` operator, which represents all dimensions as positioned in the DSD, or the keyword `series_key`, which represents all non-time dimensions as positioned in the DSD. The sorting must respect the sequence, in which the components are listed. In addition, each component, or the set of components (through the operator or keyword) can be sorted in ascending or descending order by appending `:asc` or `:desc`, with `:asc` being the default. For any component not included in the sort parameter, the related order is non-deterministic. Except for time periods, which have a natural chronological order, the sorting within a component is based on the code IDs or the non-coded component values. | | No
```

Comment:
We seem to consider timeseries data only, and thereby exclude cross-sectional data. That might be OK (not sure) but, if we want to make it more generic, we could rephrase "or the keyword `series_key`, which represents all non-time dimensions as positioned in the DSD" as follow:

"or the keyword `series_key`, which represents all dimensions, as positioned in the DSD, except the dimension at the observation level (TIME_PERIOD for timeseries data, or any dimension except TIME_PERIOD for cross-sectional data)"